src/test.rs

```diff
@@ -715,10 +716,6 @@ impl Definition {
                     continue;
                 }
 
-                if body_str.starts_with('"') && body_str.ends_with('"') {
```

Comment:
Don't know why we had this block, but we can add it back if we think it serves a purpose. It will break simple string json though (e.g. "my_body" )

Reply:
This was to fix one of the scenarios around file data loading. I agree though we shouldn't require it as long as things work otherwise, or we can come up with a better solution. This was definitely a hack :D We'll have to make sure the file data loading scenario works correctly.